Orange/widgets/data/oweditdomain.py

```diff
@@ -2527,8 +2527,16 @@ def mapper(arr, out=None, dtype=dtype, **kwargs):
 def time_parse(values: Sequence[str], name="__"):
     tvar = Orange.data.TimeVariable(name)
     parse_time = ftry(tvar.parse, ValueError, np.nan)
-    values = [parse_time(v) for v in values]
-    return tvar, values
+    _values = [parse_time(v) for v in values]
+    if np.all(np.isnan(_values)):
+        # try parsing it with pandas (like in transform)
+        dti = pd.to_datetime(values, errors="coerce")
+        _values = str_to_time(np.array(values))
```

Comment:
You extracted some code into a new function `str_to_time` to reuse it here, but then have to call `pd.to_datetime` outside of the new function anyway, to get some extra info...